Intro to css values (#44937)
mdn/content: files/en-us/web/css/reference/properties/font-width/index.md

font-width: unset;
```

### Values

> This property is specified as one of the following keyword values or a `<percentage>` value:

Reviewer: ```suggestion
This property is specified as a single value from the following list:
```
